### ISSUE TITLE
The term 'upgrade-assistant' is not recognized

### ISSUE THREAD

**Issue by LucidOcean** [dotnet/upgrade-assistant#316]
Problem encountered on https://dotnet.microsoft.com/platform/upgrade-assistant/tutorial/install-upgrade-assistant
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

upgrade-assistant : The term 'upgrade-assistant' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was
included, verify that the path is correct and try again.
At line:1 char:1
+ upgrade-assistant
+ ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (upgrade-assistant:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by mjrousos** [comment]
Hi @LucidOcean, did the install step (`dotnet tool install -g upgrade-assistant`) succeed?

**Comment by LucidOcean** [comment]
Yes, I did and it succeeded.

**Comment by mjrousos** [comment]
Thanks, @LucidOcean for the additional information. In my case, `upgrade-assistant` seems to work fine from a PowerShell prompt (which it looks like you are using) since it is found on the path. Can you check the following to make sure things are configured as expected?

1. Restart the PowerShell prompt to make sure it has up-to-date environment variables.
2. Run these commands to confirm that upgrade-assistant is installed in the expected location:
    1. `$toolsPath = [System.Environment]::ExpandEnvironmentVariables("%userprofile%") + "\.dotnet\tools"`
    2. `dir $toolsPath`      You should see 'upgrade-assistant.exe' listed in the directory. If not, that could be the problem.
3. Check that the .NET tools path is in your path environment variable by running this command:
    1. `$env:Path -split ';'`       This will produce a number of lines of output, but somewhere in them you should see the  tools path ("C:\Users\<UserName>\.dotnet\tools" or something similar). If you don't, that could explain the problem.

**Comment by LucidOcean** [comment]
Thanks for your help. 
The result of running this was entirely as expected, but I did it get it working in an odd way, explained in next post.

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        05/03/2021     10:37                .store
-a----        22/02/2021     21:06         152472 try-convert.exe
-a----        05/03/2021     10:37         143360 upgrade-assistant.exe

$env:Path -split ';' does contain my path

**Comment by LucidOcean** [comment]
To get mine working (keep in mind that everything was declaring success, from installing to path being in $env etc).

I manually edited the environment variables via windows - I changed the path to 
C:\Users\<username>\.dotnet\toolsXXX 
and saved
changed it back
C:\Users\Jonathan\.dotnet\tools
and saved

restarted vscode and it worked. I am not sure of the issue here, but it didn't work before this, even though
- I restarted powershell
- then tried restarting vscode itself
- then tried restarting my box
- a day later, after being switched off overnight, it was still not working.

I had to do the steps above which makes no sense to me... :)
